system/lib/wasmfs/backends/opfs_backend.cpp

```diff
@@ -456,7 +456,7 @@ class OPFSBackend : public Backend {
 
   std::shared_ptr<Directory> createDirectory(mode_t mode) override {
     proxy([](auto ctx) { _wasmfs_opfs_init_root_directory(ctx.ctx); });
-    return std::make_shared<OPFSDirectory>(mode, this, 0, proxy);
+    return std::make_shared<OPFSDirectory>(mode, this, 1, proxy);
```

Comment:
I wonder if we should have _wasmfs_opfs_init_root_directory return the id here?  Would that be easy?

Reply:
Yes, that would be easy.